JoinColony/colonyCDapp
# Display motions in actions list

## Description

This PR correctly displays motions in the actions list.

First, it filters out those that haven't been staked above the threshold (handled in https://github.com/JoinColony/block-ingestor/pull/53). It also filters out actions created by the voting rep extn (to avoid duplicates when finalizing a motion).

Second, it adds the correct tag to the action item.

![tags](https://user-images.githubusercontent.com/64402732/232061135-e7cd240c-554f-43a8-9e2a-89a8f18ef2b7.png)


## Testing

You need to be able to stake a motion (see #380 for instructions)

1. Create a motion. It should not appear in the actions list.
2. Stake a motion. It should only appear in the actions list if you stake above 10% total. 
3. Finalize a motion. The corresponding action should not appear in the actions list (although you'll be able to see it added to the db).
4. Create an action normally. That should appear in the actions list. 

At all times, motion state should be correctly displayed as a tag on the ActionListItem. 

If you disable the voting rep extn, we remove the tags. #357 exists to handle the display of "old" motions, i.e. ones that belong to a voting rep extn that isn't currently installed. 

**Changes** 🏗

* Update schema to include "showInActionsList" flag
* Filter actions by this flag

Resolves #302, #414
